playpen/dev-setup.sh

```diff
@@ -80,10 +80,14 @@ pushd pulp
 setfacl -m user:apache:rwx .
 popd; popd
 
+echo "Disabling strict mode for ssh"
+echo 'StrictModes no' | sudo tee -a /etc/ssh/sshd_config
+sudo systemctl restart sshd
```

Comment:
The rest of this file has been adjusted to be idempotent, as of a PR I just merged yesterday. Would you mind adding a check that this setting is not already configured before appending it to the file and restarting ssh?